tools/SourceKit/lib/SwiftLang/SwiftLangSupport.cpp

```diff
@@ -1013,9 +1000,11 @@ void SwiftLangSupport::codeComplete(
   auto options = provider->addFileSystem(FS);
   vfsOptions.options =
       llvm::make_unique<DirectlyPassedFileSystemProvider::Options>(options);
-  codeComplete(InputBuf, Offset, Consumer, Args, std::move(vfsOptions));
+  codeComplete(InputBuf, Offset, vfsOptions.options.get(), Consumer, Args,
```

Comment:
https://github.com/apple/swift/pull/28727 (fast code completion within function bodies) adds a new `OptionsDictionary *` argument (third) to `SwiftLangSupport::codeComplete`.

@marcrasi: could you please review this merge fix for `SwiftLangSupport::codeComplete`?
I simply passed `vfsOptions.options.get()` as the new `OptionsDictionary *` argument, rather than adding/propagating a new `OptionsDictionary *` argument to `SwiftLangSupport::codeComplete`.